### PR TITLE
Fix linting

### LINKED ISSUE #4: Make code comply with Python general naming conventions (lower-case function variables, Pascal case for classes, etc.)
- [ ] Some function variables using upper-case, e.g. FIELDN in get_ERA_nc_data(ncfname, FIELDN, wanted_dt, bounds). Variable used inside functions should be lower-case and words separated by underscore (e.g. snake_case). All caps is normally reserved for global defaults. e.g. a variable defined outside a function - the ERA_DIR is a good example.
- [ ] bounding_box class name should be BoundingBox (e.g. `PascalCase`)

More notes and examples on general naming conventions for python are in the dev-docs: https://github.com/ACCESS-NRI/dev-docs/wiki/Code-guidelines#naming-conventions

### LINKED ISSUE #9: Delete variables set but not used
Some variables are set but never used.
E.g. `ec_cb_file`:
https://github.com/ACCESS-NRI/replace_landsurface/blob/cdb8734c01cbab2c1dae4a09e13648c8dc4de4b8/src/replace_landsurface_with_BARRA2R_IC.py#L172

Check if the unused variables are effectively not needed and, in case, delete them.

### PR DESCRIPTION
**Pull request by atteggiani**
Fixes #4, #9, #10 and #11.

Fixed as part of the pre-commit formatting/linting issues.

**Note:**
This branch has been created starting from the [`pre-commit` branch](https://github.com/ACCESS-NRI/replace_landsurface/tree/pre-commit), which has an [open PR](https://github.com/ACCESS-NRI/replace_landsurface/pull/20).
Mark this PR as ready for review, only after checking that #20 was merged, and after rebasing this branch to `main`.

### CODE REVIEW

File: src/hres_eccb.py
Comment (penguian) [inline src/hres_eccb.py:1]: Why is this file added here as opposed to PR #17 ?

File: src/hres_ic.py
Comment (penguian) [inline src/hres_ic.py:1]: Why is this file added here as opposed to PR #17 ?

File: src/replace_landsurface_with_BARRA2R_IC.py
Comment (penguian) [inline src/replace_landsurface_with_BARRA2R_IC.py:1]: Please see PR #2 and merge your linting changes with the changes there.

File: src/replace_landsurface_with_ERA5land_IC.py
Comment (penguian) [inline src/replace_landsurface_with_ERA5land_IC.py:1]: Please see PR #2 and merge your linting changes with the changes there.

File: src/hres_eccb.py
Comment (atteggiani) [reply src/hres_eccb.py:1]: As written in the PR, this branch was created started from a different feature branch (which in turn was created from a different feature branh), therefore it also contains commits that are not relevant to this specific PR.

This is not a problem as long as we merge the "primitive" PRs first, and then always rebase the branches to `main` before merging the following PRs in order.

**Review by penguian — CHANGES_REQUESTED** [review]
Maybe wait for PR #2 to be merged, and then fix linting?

File: src/hres_ic.py
Comment (atteggiani) [reply src/hres_ic.py:1]: Duplicate of the comment above?

**Comment by atteggiani** [comment]
> Maybe wait for PR #2 to be merged, and then fix linting?

Yes that was my plan!

**Pull request closed without merging**
